### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by alivenotions** [Coding-Coach/coding-coach#292]
# Issue

## Description

The color contrast with the foreground color in the navigation links is not enough in comparison with the background color.

## Screenshot

![capture](https://user-images.githubusercontent.com/19860968/50963544-054c3680-14f3-11e9-84a5-ac4e4e0f728e.PNG)

Can also have a look at this [checker](https://webaim.org/resources/contrastchecker/?fcolor=009D6C&bcolor=69D5B1)

**Comment by alivenotions** [comment]
Won't mind working on this, but it probably requires some UX discussions first.

**Comment by emmawedekind** [comment]
I will get back to you on this.

**Comment by crysfel** [comment]
Assigned to @emmawedekind as she's our designer lead

**Comment by emmawedekind** [comment]
We can update the navigation text color to #126D4F

![image](https://user-images.githubusercontent.com/7671983/51109997-45633000-17f7-11e9-8fcc-73ae2a6cb1ff.png)

**Comment by emmawedekind** [comment]
@alivenotions Do you want to take this?

**Comment by alivenotions** [comment]
Yeah sure! Will open a PR tomorrow.

**Comment by emmawedekind** [comment]
Hi, any update @alivenotions ? :)
